bitnami/keycloak/templates/tls-secret.yaml

```
@@ -5,7 +5,7 @@ apiVersion: v1
 kind: Secret
 metadata:
   name: {{ .name }}
-  namespace: {{ $.Release.Namespace }}
+  namespace: {{ include "common.names.namespace" . | quote }}
```

In this case, we have to use `$` instead of `.`
```suggestion
  namespace: {{ include "common.names.namespace" $ | quote }}
```

---

Resolved, learning so much about helm with this PR, thank you!